websockets/handlers/simple_handshake_wsh.py

```diff
@@ -1,13 +1,25 @@
 #!/usr/bin/python
 
-from mod_pywebsocket import common, msgutil, util
-from mod_pywebsocket.handshake import hybi
+import struct
```

Comment:
not used?

Reply:
Good catch. Must have been leftover from a previous iteration.